
Setting up data pipeline
Restric
Virtual devices cannot be modified after being initialized
Epoch 1/128
2022-05-28 01:13:02.264302: I tensorflow/stream_executor/cuda/cuda_dnn.cc:366] Loaded cuDNN version 8400































































    125/Unknown - 169s 1s/step - loss: 0.2176 - accuracy: 0.9140
Traceback (most recent call last):
  File "main.py", line 58, in <module>
    model_history = model.fit(
  File "/home/melih/anaconda3/envs/tf-web/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 67, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/home/melih/Code/uni/sem6/deep_cv/train_unet.py", line 144, in on_epoch_end
    self.show_pred()
  File "/home/melih/Code/uni/sem6/deep_cv/train_unet.py", line 148, in show_pred
    _, ax = plt.subplots(3, self.sample_count)
AttributeError: 'DisplayTestCallback' object has no attribute 'sample_count'